common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -3204,6 +3204,12 @@
     ],
     "sqlState" : "42809"
   },
+  "NOT_NULL_ASSERT_VIOLATION" : {
+    "message" : [
+      "NULL value appeared in non-nullable field: <walkedTypePath>If the schema is inferred from a Scala tuple/case class, or a Java bean, please try to use scala.Option[_] or other nullable types (such as java.lang.Integer instead of int/scala.Int)."
```

Comment:
Nit?

...Path>. If

Reply:
The `<walkedTypePath>` contains newline(s) so we are OK here. I just ported it from the previous error message :)

Reply:
gotcha!